stubs/requests-oauthlib/requests_oauthlib/oauth1_auth.pyi

```diff
@@ -27,7 +28,9 @@ class OAuth1(AuthBase):
         client_class: type[Client] | None = None,
         force_include_body: bool = False,
         *,
+        realm: Incomplete | None = None,
         encoding: str = "utf-8",
         nonce: Incomplete | None = None,
         timestamp: Incomplete | None = None,
+        **kwargs: Any,
```

Comment:
Nit: We now require each `Any` to be documented.

```suggestion
        **kwargs: Any,  # passed to client_class's __init__
```

Reply:
Thanks @srittau , I've made the changes.